src/open_inwoner/openklant/migrations/0026_contactform_subject_config_default.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.2.18 on 2025-03-04 13:08
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def set_default_for_klant_config(apps, _):
+    ContactFormSubject = apps.get_model("openklant", "ContactFormSubject")
+    ESuiteKlantConfig = apps.get_model("openklant", "ESuiteKlantConfig")
+    OpenKlant2Config = apps.get_model("openklant", "OpenKlant2Config")
+
+    for contact_form_subject in ContactFormSubject.objects.filter(
+        Q(esuite_config__isnull=True) | Q(openklant_config__isnull=True)
+    ):
+        contact_form_subject.esuite_config = ESuiteKlantConfig.objects.first()
+        contact_form_subject.openklant_config = OpenKlant2Config.objects.first()
```

Comment:
I would still add this, to be on the safe side:

```suggestion
        contact_form_subject.esuite_config = contact_form_subject.esuite_config or ESuiteKlantConfig.objects.first()
        contact_form_subject.openklant_config = contact_form_subject.openklant_config or OpenKlant2Config.objects.first()
```